specification/recoveryservicessiterecovery/resource-manager/Microsoft.RecoveryServices/2016-08-10/service.json

```diff
@@ -6019,6 +6067,96 @@
       },
       "discriminator": "instanceType"
     },
+    "HealthError": {
+      "description": "Health Error",
+      "type": "object",
+      "properties": {
+        "errorSource": {
+          "description": "Source of error.",
+          "type": "string"
+        },
+        "errorType": {
+          "description": "Type of error.",
+          "type": "string"
+        },
+        "errorLevel": {
+          "description": "Level of error.",
+          "type": "string"
+        },
+        "errorCode": {
+          "description": "Error code.",
+          "type": "string"
+        },
+        "errorMessage": {
+          "description": "Error message.",
+          "type": "string"
+        },
+        "possibleCauses": {
+          "description": "Possible causes of error.",
+          "type": "string"
+        },
+        "recommendedAction": {
+          "description": "Recommended action to resolve error.",
+          "type": "string"
+        },
+        "creationTimeUtc": {
+          "format": "date-time",
+          "description": "Error creation time (UTC)",
+          "type": "string"
+        },
+        "recoveryProviderErrorMessage": {
+          "description": "DRA error message.",
+          "type": "string"
+        },
+        "entityId": {
+          "description": "ID of the entity.",
+          "type": "string"
+        },
+        "childErrors": {
+          "description": "The child health errors.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/HealthError"
+          }
+        }
+      }
+    },
+    "EventQueryParameter": {
```

Comment:
This will be used to query the list of events based on the filters provided as part of the EventQueryParameter. This will have value like startdate to enddate, severity of the filter... impacted resourcee name and so...on..